packages-common/pluggable-widgets-tools/src/typings-generator/__tests__/outputs.ts

```diff
@@ -0,0 +1,126 @@
+export const nativeResult = `/**
```

Comment:
I think it is more logical for this file to be `outputs/index.ts` Same for input - it is more logical and easier to find

Reply:
Done